ui/app/components/app/alerts/unconnected-account-alert/unconnected-account-alert.scss

```diff
@@ -7,19 +11,18 @@
     }
   }
 
-  &__footer-buttons {
+  &__footer-row {
     display: flex;
     flex-direction: row;
+  }
 
-    button:first-child {
-      margin-right: 24px;
-    }
-
-    button {
-      font-size: 14px;
-      line-height: 20px;
-      padding: 8px;
-    }
+  &__button--dismiss {
```

Comment:
Nit: `--dismiss` is not a modifier. Modifiers are meant to reflect changes in the state of a particular element (e.g. "disabled",  "error", "highlighted", "active", "focused", etc.).

The dismiss button is an _element_ in BEM terms. Something like `unconnected-account-alert__dismiss-button` would be more appropriate

Reply:
Done in: ea434da